src/engines/html5/html5.js

```diff
@@ -282,7 +288,7 @@ export default class Html5 extends FakeEventTarget implements IEngine {
    * @returns {?TextTrack} - returns the active text track element if available
    * @private
    */
-  _getSelectedTextTrack(): ?TextTrack {
+  _getSelectedTextTrackElement(): ?TextTrack {
```

Comment:
It's not an element, but a DOM object.
if it was of the format then I would agree.
```html
<video>
   <track>...</track>
</video>
```

And even for the above, you can still get the element as an object via `video.textTracks` and via DOM API selector.